administrator/components/com_admin/sql/updates/postgresql/4.0.0-2017-06-12.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `#__update_sites` ADD COLUMN `checked_out` int(10) unsigned NOT NULL DEFAULT 0;
+ALTER TABLE `#__update_sites` ADD COLUMN `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00';
```

Comment:
Should be timestamp without time zone DEFAULT '1970-01-01 00:00:00' NOT NULL for postgres